locales/en.default.json

```diff
@@ -142,7 +142,9 @@
       "volume_pricing": {
         "title": "Volume Pricing",
         "note": "Volume pricing available",
-        "minimum": "{{ quantity }}+"
+        "minimum": "{{ quantity }}+",
+        "at_price": "at",
```

Comment:
Honestly, I couldn't figure a better way. We don't want to add "at" to "price_each": "{{ price }}/ea" because as far as I understand we use "price_each" for volume-pricing and we don't want to see "at" there 🤔
<img width="382" alt="Screenshot 2023-06-29 at 10 17 13 AM" src="https://github.com/Shopify/dawn/assets/105315663/c6719fb8-e432-434c-947a-fcb4b9e90580">

Reply:
They can be 2 different strings
```
"price_at_each": "at {{ price }}/ea",
"price_each": "{{ price }}/ea"
```

Reply:
That's not actually working for me. The way I update the price per item is I use an array in JS where I add data from liquid which already looks like `$100.00 CAD/ea`.

Reply:
@eugenekasimov Wouldnt creating two strings like what Kai suggested work?

Reply:
not really, I explained why in my previous post 👆

Reply:
Okay, Kai suggested me another approach to keep my current logic and it seems working.